dome9CloudBots/bots/sql_disable_public_access.py

```diff
@@ -0,0 +1,42 @@
+# What it does: Sets "Deny public network access" Azure SQL flag to "Yes" and "Minimal TLS Version" to 1.2.
+# Usage: sql_disable_public_access
+# Example: sql_disable_public_access
+# TLS is automatically set to 1.2, if a lower version is required, change the min_tls_version variable to 1.0 or 1.1 as required

```

Comment:
Why not let the user change the tls version with param? It's much more user friendly than ask for him changing the code. also we need to support cases the user enter unsupported version

Reply:
Amended code to accept optional command line TLS parameters.